submissions/Lu-sure/memory-pair-game/index.js

```diff
@@ -0,0 +1,151 @@
+const htmlBoard = document.getElementById('board');
+const newGameButton = document.getElementById('new-game');
+const cards = [];
+const numberOfCards = 12;
+const images = [
+    {url: 'img/tree0.svg', times: 0},
+    {url: 'img/tree1.svg', times: 0},
+    {url: 'img/tree2.svg', times: 0},
+    {url: 'img/tree3.svg', times: 0},
+    {url: 'img/tree4.svg', times: 0},
+    {url: 'img/tree5.svg', times: 0}];
+const flipDelay = 800;
+const greeting = `Got 3, be smart)`;
+let score;
+
+class Card {
+    constructor(id) {
+        this.htmlEl;
+        this.htmlElInner;
+        this.frontPic = randomizeImage(images);
+        this.isOpen = false;
+        this.isMatched = false;
+        this.id = id;
+        this.render();
+    }
+
+    openCard() {
+        this.flip();
+        this.isOpen = true;
+    }
+
+    setMatched() {
+        this.htmlElInner.classList.add('hide');
+        this.isMatched = true;
+    }
+
+    flip() {
+        this.htmlElInner.classList.add('flip');
+    }
+
+    unflip() {
+        this.htmlElInner.classList.remove('flip');
+    }
+
+    render() {
+        this.htmlEl = document.createElement('div');
+        this.htmlEl.classList.add('card');
+        this.htmlEl.setAttribute('data-idx', this.id);
+
+        const cardBack = document.createElement('div');
+        const back = document.createElement('img');
+        back.setAttribute('src', 'img/back.jpg');
+        back.classList.add('pic');
+        cardBack.appendChild(back);
+        cardBack.classList.add('card-back');
+
+        const cardFront = document.createElement('div');
+        cardFront.classList.add('card-front');
+        const pic = document.createElement('img');
+        pic.classList.add('pic');
+        pic.setAttribute('src', this.frontPic);
+        cardFront.appendChild(pic);
+
+        this.htmlElInner = document.createElement('div');
+        this.htmlElInner.classList.add('card-inner');
+        this.htmlElInner.append(cardBack, cardFront);
+
+        this.htmlEl.appendChild(this.htmlElInner);
+    }
+}
+
+const htmlScore = document.createElement('p');
```

Comment:
It would be better to move this constant to the top of this script in order to keep all of the constants in one place

Reply:
Done.